fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -1995,9 +1995,12 @@ void setupSimulatedSystem(vector<Future<Void>>* systemActors,
 		       coordinatorCount);
 		ASSERT_LE(dcCoordinators, machines);
 
-		// FIXME: temporarily code to test storage cache
 		// TODO: caching disabled for this merge
 		if (dc == 0) {
+			// FIXME: we hardcode the second last (i.e. i=machines-2) machine to specifically test storage cache
+			machines++;
+
+			// FIXME: we hardcode the last (i.e. i=machines-1) machine to specifically test blob worker
```

Comment:
This would result in always one blob worker in total right? Then we'd have no fault tolerance and no testing of moving ranges back and forth.
We probably want always 2, and maybe randomly more or something.

Reply:
This is true. I'll add support for multiple (with some randomness)